iquise/utils.py

```diff
@@ -11,6 +12,9 @@
 
 from website.models import IQUISE, Donor
 
+def random_password():
+    return ''.join([random.SystemRandom().choice('abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)') for i in range(15)])
```

Comment:
any reason not to include capitals? jw

Reply:
Nope just copy pasted from elsewhere. I think the string module actually contain ascii_uppercase and ascii_lowercase to avoid retyping them all.